spec/models/movie_spec.rb

```diff
@@ -1,7 +1,30 @@
 require "rails_helper"
 
 RSpec.describe Movie, type: :model do
+  before(:each) do
+    test_data
```

Comment:
For an IC, using this test data in a model test is totally fine. If this were a collaborative project, I'd consider having your test data setup be part of some of these instance method tests in order to tell a story about what these methods do. It's a bit less readable to have the actors' data outside this file, as you can't verify behavior as easily. Again, this doesn't matter for an IC but is something to keep in mind for your group project!